src/crimson/mon/MonClient.h

```diff
@@ -119,8 +119,11 @@ class Client : public crimson::net::Dispatcher,
   void print(std::ostream&) const;
 private:
   // AuthServer methods
-  std::pair<std::vector<uint32_t>, std::vector<uint32_t>>
+  std::vector<uint32_t>
   get_supported_auth_methods(int peer_type) final;
+  std::vector<uint32_t>
+  get_supported_con_modes(int peer_type,
+			  uint32_t auth_method) final;
```

Comment:
No need to wrap lines here:
```suggestion
  std::vector<uint32_t> get_supported_auth_methods(int peer_type) final;
  std::vector<uint32_t> get_supported_con_modes(int peer_type,
			                        uint32_t auth_method) final;
```